tutorials/functional-full-stack/purescript-bridge/tutorial.md

```diff
@@ -0,0 +1,339 @@
+---
+title: Connecting a Haskell Backend to a Purescript Frontend
+published: not-yet
+ghc: 7.10.3
+purs: 0.10.5
+lts: 7.17
+libraries: purescript-bridge
+language: haskell purescript
+author-name: Javier Casas Velasco
+description: In this tutorial we will implement a way to extend the types in the Haskell backend to the Purescript frontend while maintaining consistency and simplifying communication.
+---
+# Connecting a Haskell Backend to a Purescript Frontend
+## Introduction
+At Stackbuilders we are working on a full-stack app with CollegeVine https://www.collegevine.com/ using functional languages.
+We have a Haskell backend, based on Servant, that manipulates the database and offers some endpoints to a Purescript frontend,
+that does all the React-like magic to show a really nice interface on the user's browser.
+It's great because we have advanced types, purity and all the awesome benefits that offers the Functional world.
+But not everything is perfect.
+
+### Motivation
+The problem is that we have two different codebases: one in Haskell and the other in Purescript.
+The syntax is almost the same, but that small difference means we can't directly share files.
```

Comment:
@javcasas I'm not sure what you are referring to on this statement `but that small difference`. I understand it has to be with Haskell and PureScript but maybe we need to be more explicit about what specific difference we are talking about.